jte-jsp-converter/testdata/simpleTagWithBody2/after/jte/tag/simple.jte

```diff
@@ -2,6 +2,9 @@
 
 @param String title
 
-@tag.withBody(headline = title, bodyContent = @`
+@tag.withBody(
+    headline = title,
+    bodyContent = @`
     <p>This is the body, for title ${title}</p>
-`)
\ No newline at end of file
+`
```

Comment:
The closing content backtick and the closing brace should be on the same line.

Reply:
@casid
I had an impression that closing brace should always be on a separate line, except when there are no parameters at all. Samples:
```
@tag.withoutParameters()
@tag.withOneParameter(
    param1 = "value1"
}
@tag.withContentParamNoLineBreaks(
    content1 = @`<div>Text</div>`
)
@tag.withContentParamNotEndingWithLineBreak(
    content1 = @`<div>
    <span>text</span>
</div>`
)
@tag.withContentParamEndingWithLineBreak(
    content1 = @`<div>
    <span>text</span>
</div>
`
)
@tag.withContentParamFollowedByIntParam(
    content1 = @`<div>
Text
</div>
`,
    name = "John"
)
@tag.withTwoContentParams(
    content1 = @`<span>Text 1</span>
`,
    content2 = @`<span>Text 2</span>`
)
@tag.withTwoContentParamsBothEndingWithLineBreak(
    content1 = @`<p>John</p>
`,
    content2 = @`<p>Smith</p>
`
)
```

Reply:
Hmm, especially your last example looks a bit assymetric, doesn't it?

```
@tag.withTwoContentParamsBothEndingWithLineBreak(
    content1 = @`<p>John</p>
`,
    content2 = @`<p>Smith</p>
`
)
```

This would be more consistent:

```
@tag.withTwoContentParamsBothEndingWithLineBreak(
    content1 = @`<p>John</p>
`,
    content2 = @`<p>Smith</p>
`)
```

Reply:
OK, leaving it as it is.